src/config_file.rs

```diff
@@ -256,7 +254,7 @@ pub fn save_config_db(juliaup_config_file: &mut JuliaupConfigFile) -> Result<()>
         .with_context(|| "Failed to set len to 0 for config file before writing new content.")?;
 
     serde_json::to_writer_pretty(&juliaup_config_file.file, &juliaup_config_file.data)
-        .with_context(|| format!("Failed to write configuration file."))?;
+        .with_context(|| "Failed to write configuration file.".to_string())?;
```

Comment:
Do we even need the `.to_string()`? As far as I can tell not? There are more changes like that further down that probably also don't need that `.to_string()`.

Reply:
You're right. I've removed the `.to_string()` here and a few similar places.